actions/ctf-build-image/action.yml

```diff
@@ -149,10 +156,10 @@ runs:
         echo "::endgroup::"
 
     - name: Build push docker image
-      uses: smartcontractkit/.github/actions/build-push-docker@build-push-docker/0.7.1
+      uses: smartcontractkit/.github/actions/build-push-docker@build-push-docker/v1
       with:
         context: .
-        platform: linux/amd64
+        platform: ${{ inputs.platform }}
```

Comment:
If we run an arm64 build from a amd64 runner, we're going to haove to virtualize which will be super slow.